.github/workflows/generate-feature-graphic.yml

```
@@ -0,0 +1,60 @@
+name: Generate feature graphic
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'fastlane/**'
+jobs:
+  generate-feature-graphic:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install requirements
+      run: |
+        sudo apt-get update
+        sudo apt-get install optipng
+        # Install 200 weight versions of relevant Noto (to use for languages not supported by Lexend Deca)
+        sudo apt-get install fonts-noto-extra fonts-noto-cjk-extra
+        # Custom fonts
+        mkdir "$HOME/.fonts"
+        find .scripts/generate_feature_graphic/fonts -name '*.ttf' -exec cp {} "$HOME/.fonts" \;
+        fc-cache
+    - name: Generate featureGraphic.png for each language
+      run: |
+        for lang in fastlane/metadata/android/*; do
+          pushd "$lang"
+          # Extract text after 'Catima - '
+          subtext=$(grep -oP '(?<=Catima \S ).*' title.txt || true)
+          # Generate temporary .svg with correct text
+          sed "s/Loyalty Card Wallet/$subtext/" ../../../../.scripts/generate_feature_graphic/featureGraphic.svg > featureGraphic.svg
```

```suggestion
          export subtext="$(grep -oP '(?<=Catima \S ).*' title.txt || true)"
          # Generate temporary .svg with correct text
          perl -pe 's/Loyalty Card Wallet/$ENV{subtext}/' ../../../../.scripts/generate_feature_graphic/featureGraphic.svg > featureGraphic.svg
```

You could replace `true` with `echo 'Loyalty Card Wallet'` to have an untranslated title at least. Seems better to me than just leaving it blank. idk.